packages/wonder-blocks-announcer/src/__tests__/util/dom.test.ts

```diff
@@ -0,0 +1,133 @@
+import {screen, waitFor} from "@testing-library/react";
+import {
+    createRegionWrapper,
+    createDuplicateRegions,
+    createRegion,
+    removeMessage,
+} from "../../util/dom";
+
+jest.useFakeTimers();
+jest.spyOn(global, "setTimeout");
+
+describe("Announcer utility functions", () => {
```

Comment:
**suggestion:** It would be great using AAA in these tests: https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/98402353/Unit+Testing+Best+Practices#Arrange-Act-Assert